### PR TITLE
Fabo/fix transactions not displaying for terra

### LINKED ISSUE #3412: Terra transactions only display slide details, but not main container
**Describe the bug**

For some reason, in Activity Terra transactions are displayed like empty divs, although there is no error to be seen in the browser console.

Still, the slide metada does work correctly.

image

I presume this has to do with how Terra data structure's a bit different...

### PR DESCRIPTION
**Pull request by faboweb**
Closes #3412

Should also work for emoney

**Description:**

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: changes/fabo_fix-transactions-not-displaying-for-terra [hunk 1]
```diff
@@ -0,0 +1 @@
+[Fixed] [#3412](https://github.com/cosmos/lunie/issues/3412) Show transactions on EMOney and Terra @faboweb
```
Comment (jbibla) [inline changes/fabo_fix-transactions-not-displaying-for-terra]:
```suggestion
[Fixed] [#3412](https://github.com/cosmos/lunie/issues/3412) Show transactions on EMoney and Terra @faboweb
```

File: src/components/transactions/TransactionItem.vue [hunk 2]
```diff
@@ -82,8 +82,10 @@ export default {
   }),
   computed: {
     messageTypeComponent: function() {
+      // TODO this only works for Cosmos networks. This whole logic will chang with: https://github.com/luniehq/lunie-api/issues/259
```
Comment (jbibla) [inline src/components/transactions/TransactionItem.vue]:
```suggestion
      // TODO this only works for Cosmos networks. This whole logic will change with: https://github.com/luniehq/lunie-api/issues/259
```

**Comment by faboweb** [comment]
Please just add this wording changes at your gusto in the future I will very unlikely disagree

**Comment by faboweb** [comment]
This way we can merge faster

**Pull request merged**